shell/platform/darwin/common/framework/Headers/FlutterDartProject.h

```diff
@@ -6,37 +6,67 @@
 #define FLUTTER_FLUTTERDARTPROJECT_H_
 
 #import <Foundation/Foundation.h>
+#import <TargetConditionals.h>
 
 #import "FlutterMacros.h"
 
-NS_ASSUME_NONNULL_BEGIN
```

Comment:
> I think @stuartmorgan and I may disagree on this, but I prefer `NS_ASSUME_NONNULL_BEGIN` in headers to match the way Apple recommends in their docs. https://developer.apple.com/documentation/swift/designating-nullability-in-objective-c-apis#Annotate-Regions-as-Nonnullable

I prefer to use explicit annotations, to match the way Apple recommends in their docs: https://developer.apple.com/documentation/swift/designating-nullability-in-objective-c-apis#Annotate-Nullability-of-Individual-Declarations 😉 

(More seriously, I'm not aware of any actual recommendation by Apple to use one vs the other; on that page they document both and don't seem to indicated a preferred version. They do say you can "simplify the process" the process using the macros; but the core argument from the anti-macro camp, of which I am a member, is that simplifying the *process* at the expense of the correctness of the *result* of the process is not a good tradeoff, and the problem is right in the macro name: "assume".)

> I don't want to start moving the other way in the iOS embedder since this is now in `common` and would be shared.

We are agreed on this; every other header in the directory uses the macros currently, so this header should as well; inconsistency actually makes the problem of the macros *worse* than using the macros everywhere, so if there were to be a change to the pattern for `/common/`, it should be done as a whole in its own PR, not incrementally.

Reply:
Done